test/unittests/pydra/test_interfaces.py

```diff
@@ -0,0 +1,44 @@
+import pytest
+from pydra.tasks.nipype1.utils import Nipype1Task
+
+from clinica.pydra.interfaces import (
+    CAPSFileDataGrabber,
+    CAPSGroupDataGrabber,
+    caps_reader,
+)
+from clinica.pydra.query import BIDSQuery, CAPSFileQuery, CAPSGroupQuery
+
+
+def test_bids_reader(tmp_path):
+    from nipype.interfaces.io import BIDSDataGrabber
+
+    from clinica.pydra.interfaces import bids_reader
+
+    task = bids_reader(BIDSQuery({}), tmp_path)
```

Comment:
I am not sure what a `BIDSQuery({})` represents, tbh. I would expect this to be an invalid value and therefore throw at construction. Assuming the goal is to represent a default `BIDSQuery`, then the latter should rather be constructed with `BIDSQuery()` imo.